lib/resources/etc_libuser.rb

```diff
@@ -0,0 +1,108 @@
+require 'utils/simpleconfig'
+require 'utils/object_traversal'
+
+module Inspec::Resources
+  class EtcLibUser < Inspec.resource(1)
+    name 'etc_libuser'
+    desc 'Use the etc_libuser InSpec audit resource to check the configuration of /etc/libuser.conf.
+          Due to limitations of the RSpec engine and for consistency, please use the method described here
+          (https://github.com/chef/inspec/issues/1543) for accessing the hierarchical values in
+          /etc/libuser.conf'
+    example "
+          ---start sample /etc/libuser.conf---
+          # See libuser.conf(5) for more information.
+          ...
+          [import]
+          # Data from these files is used when libuser.conf does not define a value.
+          # The mapping is documented in the man page.
+          login_defs = /etc/login.defs
+          default_useradd = /etc/default/useradd
+          ...
+          ---end sample /etc/libuser.conf---
+
+          describe etc_libuser do
+            its(['import','login_defs']){should eq ['/etc/login.defs']}
+          end
+         "
+
+    include ObjectTraverser
+    attr_reader :params, :raw_content
+
+    class UnknownEtcLibUserConf < StandardError; end
+
+    def initialize(conf_path = nil)
+      @conf_path = conf_path || inspec.os_env('LIBUSER_CONF').content || '/etc/libuser.conf'
```

Comment:
Can you provide a use case for grabbing the path via an environment variable? This is not standard in our resources and is not something we advocate for a variety of reasons.

Reply:
The reason that I added it was because the man page for /etc/libuser.conf (https://linux.die.net/man/5/libuser.conf) states:

"Files
/etc/libuser.conf The default location of the configuration file. Can be overridden by the LIBUSER_CONF environment variable, except in set-uid or set-gid programs."